SeaPublicWebsite/Views/Cookie/CookieDetails.cshtml

```diff
@@ -1,75 +1,96 @@
-                    <td class="govuk-table__cell">2 years</td>
+                    <td class="govuk-table__cell">@SharedLocalizer["_ga This helps us count how many people visit our service by tracking if you’ve visited before"]</td>
+                    <td class="govuk-table__cell">@SharedLocalizer["2 years"]</td>
                 </tr>
                 <tr class="govuk-table__row">
                     <td class="govuk-table__cell">_gid</td>
-                    <td class="govuk-table__cell">This helps us count how many people visit our service by tracking if you’ve visited before</td>
-                    <td class="govuk-table__cell">24 hours</td>
+                    <td class="govuk-table__cell">@SharedLocalizer["_ga This helps us count how many people visit our service by tracking if you’ve visited before"]</td>
+                    <td class="govuk-table__cell">@SharedLocalizer["24 hours"]</td>
                 </tr>
                 <tr class="govuk-table__row">
                     <td class="govuk-table__cell">_gat</td>
-                    <td class="govuk-table__cell">This helps Google to manage the rate at which they track your site interactions (when high traffic occurs)</td>
-                    <td class="govuk-table__cell">1 minute</td>
+                    <td class="govuk-table__cell">@SharedLocalizer["_gat This helps Google to manage the rate at which they track your site interactions (when high traffic occurs)"]</td>
+                    <td class="govuk-table__cell">@SharedLocalizer["1 minute"]</td>
                 </tr>
                 <tr class="govuk-table__row">
                     <td class="govuk-table__cell">_gat_govuk_shared</td>
-                    <td class="govuk-table__cell">This helps Google to manage the rate at which they track your site interactions (when high traffic occurs)</td>
-                    <td class="govuk-table__cell">1 minute</td>
+                    <td class="govuk-table__cell">@SharedLocalizer["_gat_govuk_shared This helps Google to manage the rate at which they track your site interactions (when high traffic occurs)"]</td>
+                    <td class="govuk-table__cell">@SharedLocalizer["1 minute"]</td>
                 </tr>
             </tbody>
         </table>
 
-        <h3 class="govuk-heading-s">Cookies message</h3>
-        <p class="govuk-body">You may see a banner when you visit our service inviting you to accept cookies or review your settings. We’ll set cookies so that your computer knows you’ve seen it and not to show it again, and also to store your settings.</p>
+        <h3 class="govuk-heading-s">@SharedLocalizer["Cookies message"]</h3>
+        <p class="govuk-body">@SharedLocalizer["You may see a banner when you visit our service inviting you to accept cookies or review your settings. We’ll set cookies so that your computer knows you’ve seen it and not to show it again, and also to store your settings."]</p>
```

Comment:
It seems a bit strange to use the English text as the resource key - especially when it's this long. I can see that it might be convenient when reading the code, but it might go out of date with the actual translation text.

Reply:
Agreed, this is something that will perhaps have to be changed in a separate branch. 
My original thought behind this approach to resource keys was flawed, unfortunately. I would also reconsider the balance between shared resources vs separate resource files.